packages/server/modules/webhooks/graph/schemas/webhooks.graphql

```diff
@@ -0,0 +1,76 @@
+extend type Stream {
+  webhooks(id: String): WebhookCollection
+    @hasRole(role: "stream:owner")
```

Comment:
This directive can only be used for checking against the `server_acl`, it doesn't work for named resources. Hence, it should be `@hasRole(role: "server:user")` and resource access should be checked inside the resolver (like we do everywhere else). 

Currently, this prevents anyone but server admins to create webhooks on a stream, which probably explains why it passes tests (which probably will need some updating?).

Reply:
totally my bad! soz for the fuckup 😅
✔ fixed